flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/utils/TableTestBase.scala

```diff
@@ -1071,4 +1064,37 @@ object TableTestUtil {
     createTableMethod.setAccessible(true)
     createTableMethod.invoke(tEnv, queryOperation).asInstanceOf[Table]
   }
+
+  def readFromResource(path: String): String = {
+    val inputStream = getClass.getResource(path).getFile
+    val fullContent = Source.fromFile(inputStream).mkString
+    val license =
+      """/*
+        | * Licensed to the Apache Software Foundation (ASF) under one
+        | * or more contributor license agreements.  See the NOTICE file
+        | * distributed with this work for additional information
+        | * regarding copyright ownership.  The ASF licenses this file
+        | * to you under the Apache License, Version 2.0 (the
+        | * "License"); you may not use this file except in compliance
+        | * with the License.  You may obtain a copy of the License at
+        | *
+        | *     http://www.apache.org/licenses/LICENSE-2.0
+        | *
+        | * Unless required by applicable law or agreed to in writing, software
+        | * distributed under the License is distributed on an "AS IS" BASIS,
+        | * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        | * See the License for the specific language governing permissions and
+        | * limitations under the License.
+        | */
+        |""".stripMargin
+    fullContent.replace(license, "")
```

Comment:
We can add an exclusion to parent [pom.xml](https://github.com/apache/flink/blob/master/pom.xml#L1336) to avoid introducing license header and this method.